Formula/rubyfmt.rb

```diff
@@ -0,0 +1,12 @@
+class Rubyfmt < Formula
+  desc "Ruby autoformatter"
+  homepage "https://github.com/penelopezone/rubyfmt"
+  url "https://github.com/penelopezone/rubyfmt/archive/v0.2.0.zip"
+  sha256 "1b4314240f817e54a1bd6d600279121ec97c7f0aeb1e334bffe264590d652e34"
+
+  def install
+    system "make"
+    bin.install "build/rubyfmt.rb" => "rubyfmt"
+  end
+end
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.